### PR TITLE
Donation Banner Thermometer

### PR DESCRIPTION
**Pull request by jbuckner**
This extract the donation form thermometer to its own component.

Note: The base branch in this MR is the OpenWC scaffolding to eliminate those files from the MR.

banner_thermometer

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,6 @@ language: node_js
 node_js:
   - node
 
-# Only run travis when pushing to master branch (and PRs)
-branches:
-  only:
-    - master
```
Comment (jbuckner) [inline .travis.yml:8]: This MR is into the initial scaffolding branch and this was causing the tests to not run

**Review by iisa — APPROVED** [review]
awesome!

File: packages/banner-thermometer/demo/index.html [hunk 2]
```diff
@@ -3,24 +3,178 @@
...
       html`
-        <banner-thermometer .title=${title}>
-          some more light-dom
-        </banner-thermometer>
+        <div class="thermometer-container">
+          <donation-banner-thermometer id="thermometer" currentAmount=${defaultCurrentAmount} goalAmount=${defaultGoal}>
+          </donation-banner-thermometer>
+        </div>
+
+        <fieldset>
```
[the remark this answers is not in the document]
Comment (jbuckner) [reply packages/banner-thermometer/demo/index.html:116]: Every time I've tried using Storybook, I spend so much time trying to get the configuration to work that I just give up.

[the remark this answers is not in the document]
Comment (shaneriley) [reply packages/banner-thermometer/demo/index.html:116]: 👍  to that sentiment. I feel like you really need to understand the inner workings of Storybook to use it in this sort of context.

**Pull request merged**